packages/ckeditor5-dev-tests/tests/utils/manual-tests/compilehtmlfiles.js

```diff
@@ -10,46 +10,94 @@
 const mockery = require( 'mockery' );
 const sinon = require( 'sinon' );
 const path = require( 'path' );
-const commonmark = require( 'commonmark' );
-const fs = require( 'fs-extra' );
 const chokidar = require( 'chokidar' );
 const gutil = require( 'gulp-util' );
 
-describe( 'compileHtmlFiles', () => {
-	let sandbox = sinon.sandbox.create();
+const fakeDirname = path.dirname( require.resolve( '../../../lib/utils/manual-tests/compilehtmlfiles' ) );
 
-	let files = {};
+describe( 'compileHtmlFiles', () => {
+	let sandbox, stubs, files, compileHtmlFiles;
 	let patternFiles = {};
-	const fakeDirname = path.dirname( require.resolve( '../../../lib/utils/manual-tests/compilehtmlfiles' ) );
-
-	const mdParserSpy = sandbox.spy();
-	const htmlRendererSpy = sandbox.spy( () => '<h2>Markdown header</h2>' );
-	const getRelativeFilePathSpy = sandbox.spy( pathToFile => pathToFile );
 
-	before( () => {
+	beforeEach( () => {
 		mockery.enable( {
+			useCleanCache: true,
 			warnOnReplace: false,
 			warnOnUnregistered: false
 		} );
-		mockery.registerMock( '../getrelativefilepath', getRelativeFilePathSpy );
-		mockery.registerMock( '../glob', ( pattern ) => patternFiles[ pattern ] );
-		mockery.registerMock( 'dom-combiner', ( ...args ) => args.join( '\n' ) );
+
+		sandbox = sinon.sandbox.create();
```

Comment:
I would say - KISS. 

As you said above: "_Lets do not overcomplicate our problem with tests_" :D